base_classes/NXdisk_chopper.nxdl.xml

```diff
@@ -42,37 +57,65 @@
 		</enumeration>
 	</field>
 	<field name="rotation_speed" type="NX_FLOAT" units="NX_FREQUENCY">
-		<doc>chopper rotation speed</doc>
+		<doc>
+			Chopper rotation speed. Positive for anticlockwise rotation when
+			facing the sample, negative otherwise.
+		</doc>
 	</field>
 	<field name="slits" type="NX_INT">
 		<doc>Number of slits</doc>
 	</field>
 	<field name="slit_angle" type="NX_FLOAT" units="NX_ANGLE">
-		<doc>angular opening</doc>
+		<doc>Angular opening</doc>
 	</field>
 	<field name="pair_separation" type="NX_FLOAT" units="NX_LENGTH">
-		<doc>disc spacing in direction of beam</doc>
+		<doc>Disk spacing in direction of beam</doc>
+	</field>
+	<field name="slit_edges" type="NX_FLOAT" units="NX_ANGLE">
+		<doc>
+			Angle of the each edge of each slit from the position of the
+			top-dead-center timestamp sensor, anticlockwise when facing the sample.
+			The first edge must be the opening edge of a slit, thus the last edge
+			may have an angle greater than 360 degrees.
+		</doc>
+		<dimensions><dim index="1" value="2n"/></dimensions>
+	</field>
+	<field name="top_dead_center" type="NX_NUMBER" units="NX_TIME">
```

Comment:
We should allow a start_time attribute (or whatever is consistent with NXlog).

Reply:
:+1: done